docs/pages/billing/usage-based-pricing.mdx

```diff
@@ -0,0 +1,160 @@
+
+Monthly updated users reflect the number of unique users who download an update in a billing cycle. At the same time, global edge bandwidth represents the total amount of bandwidth used for all update downloads. Moreover, each monthly updated user includes 40 MiB of global edge bandwidth, up to that plan's global edge bandwidth limit.
+
+> **Note**: A monthly updated user counts only once per billing cycle, regardless of how many updates this user downloads.
+
+Each plan has a number of monthly updated users and global edge bandwidth included as part of the subscription. These differ for each plan and the most updated numbers. See our [Pricing page](https://expo.dev/pricing), for more information.
+
+> **Note**: The [On-demand plan](/billing/plans/#on-demand) includes the same amount of monthly updated users and global edge bandwidth as the Free plan.
+
+For example, let's consider a subscriber to the On-demand plan who deploys 5 updates of 20 MiB each via EAS Update to 10,000 users. The subscription to the plan includes 1,000 monthly updated users and 100 GiB per month. As a result, the subscriber's overage bill will be:
+
+| Description           | Price           | Quantity | Total |
+| --------------------- | --------------- | -------- | ----: |
+| Updated users         | $0.005 per user | 9,000    |   $45 |
+| Global edge bandwidth | $0.10 per GiB   | 525 GiB  | $52.5 |
+| **Total**             |                 |          | $97.5 |
+
+Out of the 10,000 users, 1,000 are included in the On-demand plan. As a result, 9,000 are billed for as part of usage-based billing. Paying for 9,000 updated users also includes approximately 351.6 GiB (9000 users \* 40 MiB / 1024).
+
+The global edge bandwidth calculation is:
+
+| Description                                       | Calculation                         | Quantity     |
+| ------------------------------------------------- | ----------------------------------- | ------------ |
+| Bandwidth used to send updates                    | 5 updates \* 20 MiB \* 10,000 users | 976.5625 GiB |
+|                                                   |
+| Bandwidth included in plan                        |                                     | 100 GiB      |
+| Bandwidth included with 9,000 extra updated users | 9,000 \* 40 MiB                     | 351.5625 GiB |
+| **Total**                                         | 976.5625 - 100 - 351.5625           | 525 GiB      |
+
+> **Note**: If the subscriber then sends another 20 MiB update to the same 10,000 users in the current billing cycle, they will only pay for any bandwidth overage used. This is because Expo only charges for [unique monthly updated users](/eas-update/faq/#how-are-monthly-updated-users-counted-for-a-billing-cycle).
+
+If the same subscriber is on a Production plan, they will pay $0 as the Production plan includes 20,000 monthly updated users and 1 TiB (1024 GiB). As such, there is no overage.
```

Comment:
```suggestion
> **Note**: If the subscriber then sends another 20 MiB update to the same 10,000 users in the current billing cycle, they will only pay for any extra bandwidth used. This is because Expo only charges for [unique monthly updated users](/eas-update/faq/#how-are-monthly-updated-users-counted-for-a-billing-cycle).

If the same subscriber is on a Production plan, they will pay $0 as the Production plan includes 20,000 monthly updated users and 1 TiB (1024 GiB). As such, there is no extra bandwidth usage.
```